pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -398,10 +399,21 @@ func (r *ReconcilePolicyRecommendation) Reconcile(ctx context.Context, request r
 		certificateManager.AddToStatusManager(r.status, helper.InstallNamespace())
 
 		if !r.multiTenant {
-			// Zero-tenant and single tenant setups install resources inside tigera-policy-recommendation namespace. Thus,
-			// we need to create a tigera-ca-bundle inside this namespace in order to allow communication with Linseed
-			trustedBundleRW = certificateManager.CreateTrustedBundle(managerInternalTLSSecret, linseedCertificate)
+			// Zero-tenant and single tenant setups install resources inside calico-system namespace. Thus,
+			// we need to create a tigera-ca-bundle (named policy-recommendation-ca-bundle) inside this namespace in order to allow communication with Linseed
+			trustedBundleRW, err = certificateManager.CreateNamedTrustedBundleFromSecrets(ResourceName, r.client,
+				helper.TruthNamespace(), false)
+			if err != nil {
+				r.status.SetDegraded(operatorv1.ResourceCreateError, "Unable to create the trusted bundle", err, logc)
```

Comment:
It isn't really a `ResourceCreateError`, as it can only error on file read problems, right before making the pem file, maybe  `CertificateError` best encapsulates the error at this time?